src/test/java/com/epam/ta/reportportal/dao/suite/TestItemRepositoryTest.java

```diff
@@ -57,20 +61,19 @@ public void streamItemIdsTest() {
 
 		List<Long> ids = stream.collect(Collectors.toList());
 
-		Assert.assertTrue(CollectionUtils.isNotEmpty(ids));
-		Assert.assertEquals(5, ids.size());
+		assertTrue(CollectionUtils.isNotEmpty(ids));
```

Comment:
what do you think about using such comments like below? would it be more readable with such commnets? 
https://github.com/reportportal/commons-dao/blob/61488504b441ff9ca5e9ff8eed1493ef7609a4ad/src/test/java/com/epam/ta/reportportal/dao/suite/UserRepositoryTest.java#L102-L108

Reply:
i meant 
//given
//when
//then